OracleSiebel/OEL/build/multi-stage/sai/dockerfile

```diff
@@ -0,0 +1,108 @@
+# LICENSE UPL 1.0
+#
+# Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+#
+# Since: Dec, 2019
+# Authors: andreas.thieltges@oracle.com and duncan.ford@oracle.com
+# Description: Build Siebel SAI container
+#
+# Notes: This build is execute from the build-all-multistage script
+#
+# DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
+
+# Globals - these are overridden by the calling script
+
+ARG VERSION=19.11
+ARG BUILD_DATE=2019-12-06
+
+
+FROM oraclelinux:latest as temp
```

Comment:
Please switch to `oraclelinux:7-slim` (preferred) or at least `oraclelinux:7` as `oraclelinux:latest` will switch to Oracle Linux 8 fairly soon

Reply:
Done